### PR TITLE
Adds roadmap blog

### PR DESCRIPTION
**Pull request by elfisher**
### Description
Adds a blog to highlight the proposed release schedule updates

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by elfisher** [comment]
@stockholmux I did this off of main, but there seems to be a ton of commits. Should this be based off of the prod branch?

**Comment by stockholmux** [comment]
@elfisher It _realllllly_ looks like you made a branch off of `prod` and tried to push it into `main`. If you look at any of the commits that got pulled in they are just the merge commits that are used to trigger a production build.

I could go forward from here as these commits don't change anything but it's overwhelmed the DCO checker as it can't accept this many commits at once 😂... never seen that before!

I would just close the commit out, sync your main branch with the remote main branch and make you single file change and do another commit.

**Comment by elfisher** [comment]
> @elfisher It _realllllly_ looks like you made a branch off of `prod` and tried to push it into `main`. If you look at any of the commits that got pulled in they are just the merge commits that are used to trigger a production build.
> 
> I could go forward from here as these commits don't change anything but it's overwhelmed the DCO checker as it can't accept this many commits at once 😂... never seen that before!
> 
> I would just close the commit out, sync your main branch with the remote main branch and make you single file change and do another commit.

Sounds good! I'll give that a go and push that up accordingly.

**Comment by elfisher** [comment]
@stockholmux would it work to publish this tomorrow (02/22)?

File: _posts/2022-02-09-roadmap-proposal.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### Considerations
+
+Before we get to the dates themselves, here are the things I was thinking about when I picked them. 
+
+**1) The OpenSearch Project uses semver.** 
+As a reminder, the OpenSearch project assigns version numbers to releases following [the semantic versioning convention](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/).  This means we won’t introduce an incompatible change without also incrementing the major version number.  
+
+**2) We’re excited about Lucene 9.0.0****.**
+Lucene 9.0.0 [released](https://lucene.apache.org/core/corenews.html#apache-lucenetm-900-available) on December 7, 2021. It includes several new features and performance improvements we would like to make available to users of OpenSearch, including K-NN support, Vectors, Big Endian, Faster numeric indexing, faster sorting, concurrent merge scheduler, and prototype java jigsaw module support. It will take a few releases for OpenSearch to take full advantage of it, but we’re so excited by the potential that we wanted to get Lucene 9.0.0 support in as soon as possible rather than try and bundle it with other changes.
```
Comment (stockholmux) [inline _posts/2022-02-09-roadmap-proposal.md:27]: 'java jigsaw' -> 'Java Jigsaw'

**Review by stockholmux — CHANGES_REQUESTED** [review]
Comments inline for markup and grammar edits.

I generally like the post. For content, I think one thing might be a bit odd. We talk about 'SemVer' and then two points later we talk about tent pole for major versions. Those aren't incompatible, but it's an incomplete picture. In SemVer major versions also represent oopsies - something we may have to fix that change the API. From the description in this blog post, it's not entirely clear that this could happen. Very real possibilities exist that we could rev a major version without these tent-pole features (example: zero day in Node.js that could only be resolved with a change to a new Node.js major would result in a major version rev, as an example).

**Review by stockholmux — APPROVED** [review]
LGTM - I'll change the date on the backend.

**Pull request merged**